docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tudorg/xgo-deb6-1.4.2
```

Comment:
At least I don't know this guy: https://hub.docker.com/r/elastic/ I doesn't look like he needs it. Should we ask him?
Beat is taken: https://hub.docker.com/r/beat/
Beats is available: https://hub.docker.com/r/beats/